src/contracts/pods/EigenPod.sol

```diff
@@ -92,7 +96,18 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     uint256 public nonBeaconChainETHBalanceWei;
 
      /// @notice This variable tracks the total amount of partial withdrawals claimed via merkle proofs prior to a switch to ZK proofs for claiming partial withdrawals
-    uint64 public sumOfPartialWithdrawalsClaimedGwei;
+    uint256 public sumOfPartialWithdrawalsClaimedWei;
```

Comment:
noting that this change is **totally upgrade incompatible**.
might end up being OK depending on rollout plans though :/

Reply:
oh shoot ur right we already had this in the contract, fuck I forgot

Reply:
ah I forgot we upgraded the contract rip.  We didn;t on mainnet tho right?

Reply:
ok just checked, we haven't on mainnet